### PR TITLE
Separate command example and output in install.md to ease copying

### LINKED ISSUE #1290: Separate shell command example and example output in docs
This issue is in reference to the bug that was pointed out in #1205 , related to separating the command example and output, for easy copying and more readability!

Referring to the conversation [here](https://github.com/k0sproject/k0s/pull/1220#issuecomment-953531362), as there are many pages in the documentation where this bug can be improved/updated, i feel creating separate issues for each & making them as `good-first-issues` would be a nice way for new folks in the community to contribute to!

Hoping, i can proceed with making separate issues like this, for each page of the documentation 🙂
Would love your thoughts & suggestions on this!!

### PR DESCRIPTION
**Pull request by verma-kunal**
Signed-off-by: Kunal Verma <vkunal321@gmail.com>

---
name: Pull Request
about: Create a Pull Request
title: ''
labels: ''
assignees: ''

---

**Issue**
Fixes #1290 

**What this PR Includes**
Fixes a small bug, that would allow to only copy the particular command from the documentation & not the generated output

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ Though the Quick Start material is written for Debian/Ubuntu, you can use it for
 
     ```shell
     $ sudo k0s status
```
Comment (verma-kunal) [inline docs/install.md:50]:
```suggestionsudo k0s status
```

Comment (verma-kunal) [inline docs/install.md:50]:
```suggestion
    sudo k0s status
```

File: docs/install.md [hunk 2]
```diff
@@ -63,6 +66,9 @@ Though the Quick Start material is written for Debian/Ubuntu, you can use it for
 
     ```shell
     $ sudo k0s kubectl get nodes
```
Comment (verma-kunal) [inline docs/install.md:68]:
```suggestion
    sudo k0s kubectl get nodes
```

**Comment by kke** [comment]
Needs a rebase.

> Fixes a small bug, that would allow to only copy the particular command from the documentation & not the generated output

It is intentional to split the command and the output so that you can use the generated copy-button to copy just the command for trying it yourself without getting the output to clipboard.

**Review by kke — CHANGES_REQUESTED** [review]
Needs a rebase.

Also:

>Fixes a small bug, that would allow to only copy the particular command from the documentation & not the generated output

It is intentional to split the command and the output so that you can use the generated copy-button to copy just the command for trying it yourself without getting the output to clipboard.

**Comment by soider** [comment]
@verma-kunal needs to be rebased over the main branch

**Pull request closed without merging**